parquet-column/src/main/java/org/apache/parquet/schema/PrimitiveComparator.java

```diff
@@ -206,6 +206,31 @@ public String toString() {
         }
       };
 
+  /*
+   * This comparator is for comparing two timestamps represented as int96 binary.
+   */
+  static final PrimitiveComparator<Binary> BINARY_AS_INT_96_COMPARATOR = new BinaryComparator() {
```

Comment:
Shouldn't this be `AS_INT96_TIMESTAMP`? This is not a comparison that would work for int96.

Reply:
INT96 is a physical type and there was never a converted/logical type for it. Ergo it is implicitly only a timestamp.

Adding `AS_INT96_TIMESTAMP` is redundant but perhaps leaves some context for the reader on what we are comparing here.